_sass/_base.scss

```diff
@@ -53,6 +53,12 @@ a {
   }
 }
 
+h1, h2, h3, h4, h5, h6 {
+  a {
+    text-decoration: none;
```

Comment:
Blerg, I wanted to hyperlink the blog post titles to their detail pages, but this meant removing the `text-decoration: underline` from the default styling of links. Not sure if this is super good UX or not, since it means it won't be obvious that the headings are clickable. BUT, at least we're also adding "read more" links (with additional screen-reader-only text) below the excerpts to provide more explicit links...